app/services/bill-runs/two-part-tariff/fetch-licences.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+/**
+ * Fetches 2PT Licences for the matching region and billing period
+ * @module FetchLicencesService
+ */
+
+const FetchChargeVersionsService = require('./fetch-charge-versions.service.js')
+
+/**
+ * Fetches 2PT Licences for the matching region and billing period plus the charge versions associated with them grouped
+ * by licence
+ *
+ * @param {*} regionId UUID of the region being billed that the licences must be linked to
+ * @param {*} billingPeriod Object with a `startDate` and `endDate` property representing the period being billed
+ *
+ * @returns {Object} Contains an array of unique licence IDs and array of charge versions to be processed
```

Comment:
We can be explicit with the types in the JSDoc. Plus I'm not sure the return comment is correct anymore.

```suggestion
 * @param {String} regionId UUID of the region being billed that the licences must be linked to
 * @param {Object} billingPeriod Object with a `startDate` and `endDate` property representing the period being billed
 *
 * @returns {Object[]} the licences to be matched, each containing an array of charge versions applicable for two-part tariff
```